### DISCUSSION TITLE
run command without starting environement

### DISCUSSION THREAD

**Discussion by PeterSchuhmannNetflow** [ddev/ddev#4550] [Q&A] [answered]
i want to build a custom command to setup ddev like a custom version of ddev config. But every time i run a custom command the environment gets started. That does not happen with ddev config. Has anyone an idea on how to do this?

I could use a regular command but the idea is to modify the config any further and with the regular commands the changes will happen after the environment started already. That means it will apply after a restart. 

i am thankful for any hints.

**Comment by rfay** [comment, marked as answer]
You'll probably be interested in 
* https://github.com/drud/ddev/issues/2688

But you can just use any script, include the script in your project. A ddev custom command is just a script anyway...

**Reply by PeterSchuhmannNetflow** [reply]
thx. didn't see that one on my research. sorry about that. but that is exactly what i am looking for.
